include/userobjects/MoabSkinner.h

```diff
@@ -118,6 +120,31 @@ class MoabSkinner : public GeneralUserObject
                               const unsigned int & density_bin,
                               const unsigned int & subdomain_bin) const;
 
+  /**
+   * Whether the skinner builds a graveyard
+   * @return whether a graveyard is built
+   */
+  virtual const bool & hasGraveyard() const { return _build_graveyard; }
```

Comment:
For my understanding, the case in which we don't want the skinner to build a graveyard is if the boundary conditions are picked up from the MOOSE mesh?

Reply:
For the current implementation, we always want the skinner to build a graveyard. This function is used to detect if the user set `build_graveyard` on the `MoabSkinner`. If the user forgot it, then instead of erroring out we set the graveyard flag to true from inside `OpenMCCellAverageProblem`. So basically, this is just being more forgiving if the user forgot to turn the graveyard feature on, we do it for them under the hood.

Reply:
Got it. Then I guess maybe my question above relates back to Aurora and is more of a @helen-brooks question.

Reply:
Right, I think longer term we would not want the graveyard if MOOSE is setting up reflective/periodic/etc. BCs for us.

Reply:
Right on. Not something to hold up this PR over by any means. I can find out more later 👍🏻